pkg/integrationteststatus/integration_test_status.go

```diff
@@ -33,6 +33,8 @@ const (
 	IntegrationTestStatusPending IntegrationTestStatus = iota + 1 // Pending
 	// Starting to handle an integration test scenario for a snapshot
 	IntegrationTestStatusInProgress // InProgress
+	// Integration PLR cancelled for this ITS and snapshot
+	IntegrationTestStatusCancelled // Cancelled
```

Comment:
And because you added new state, we need followup story to update status reporters in pull requests

Reply:
Created: [STONEINTG-668](https://issues.redhat.com/browse/STONEINTG-668)